qiskit_nature/optionals.py

```diff
@@ -0,0 +1,90 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Additional optional constants.
+"""
+
+from typing import Optional, Callable
+from shutil import which
+from qiskit.utils import LazyDependencyManager, LazyImportTester, LazySubprocessTester
+
+
+class NatureLazyCommandTester(LazyDependencyManager):
+    """
+    A lazy checker that a command-line tool is available. This just checks if the command is
+    not empty.
+    """
+
+    __slots__ = ("_command",)
+
+    def __init__(
+        self,
+        command: str,
+        *,
```

Comment:
:slightly_smiling_face:

Reply:
If it is about the formatting, it is `black`s formatter fault  :-)

Reply:
No it's about [PEP 3102](https://www.python.org/dev/peps/pep-3102/) :+1: